man/coap_session.txt.in

```diff
@@ -60,6 +63,8 @@ const coap_session_t *_session_);*
 
 *int coap_session_get_ifindex(const coap_session_t *_session_);*
 
+*coap_endpoint_t *coap_session_get_endpoint(const coap_session_t *_session_);*
+
```

Comment:
This should be before `coap_session_get_ifindex()` in the same order as the NAME list is defined.